subscriptions-tests/src/test/java/io/arlas/subscriptions/DataSetTool.java

```diff
@@ -274,21 +260,20 @@ public static void loadSubscriptions(boolean loadMongo, boolean createArlas) thr
             MongoDatabase mongoDatabase = mongoDBFactoryConnection.getClient().getDatabase(mongoDBname);
             mongoCollection = mongoDatabase.getCollection(MongoUserSubscriptionDAOImpl.ARLAS_SUBSCRIPTION_DB_NAME, UserSubscription.class);;
             mongoCollection.insertOne(subscription);
-            client.prepareIndex(DataSetTool.SUBSCRIPTIONS_INDEX_NAME, DataSetTool.SUBSCRIPTIONS_TYPE_NAME, subscription.getId())
-                    .setSource(mapper.writeValueAsString(indexedUserSubscription), XContentType.JSON).get();
+            client.index(DataSetTool.SUBSCRIPTIONS_INDEX_NAME, subscription.getId(), mapper.writeValueAsString(indexedUserSubscription));
         }
     }
 
-    private static void createIndex(String indexName, String typeName, String mappingFileName) throws IOException {
-        String mapping = getString(DataSetTool.class.getClassLoader().getResourceAsStream(mappingFileName));
+    private static void createIndex(String indexName, String typeName, String mappingFileName) throws IOException, ArlasException {
+        String mapping = IOUtils.toString(new InputStreamReader(DataSetTool.class.getClassLoader().getResourceAsStream(mappingFileName)));
```

Comment:
Not sure that you need the `InputStreamReader`

Reply:
Actually the IOUtils.toString method requires a Reader.